pkg/bindings/kube/kube.go

```diff
@@ -0,0 +1,97 @@
+package kube
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/containers/image/v5/types"
+	"github.com/containers/podman/v4/pkg/auth"
+	"github.com/containers/podman/v4/pkg/bindings"
+	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/sirupsen/logrus"
+)
+
+func Play(ctx context.Context, path string, options *PlayOptions) (*entities.PlayKubeReport, error) {
```

Comment:
For consistency reasons I don't like using the old `PlayKubeReport` but use a new `KubePlayReport`. 

[Type aliasing](https://yourbasic.org/golang/type-alias/) should do the trick.